### PR TITLE
Updating calculator to use GOV.UK elements

### PR DESCRIPTION
**Pull request by andysellick**
* Landing page updated
* Calculator page form elements and validation use GOV.UK Elements

*Old*
screen shot 2017-06-05 at 10 19 24

*New*
calculators-form
calculators-errors

### CODE REVIEW

File: app/views/child_benefit_tax/main.html.erb [hunk 1]
```diff
@@ -50,7 +50,7 @@
         </div>
       </fieldset>
 
-      <fieldset id="is-part-year-claim">
+      <fieldset id="is_part_year_claim">
```
Comment (fofr) [inline app/views/child_benefit_tax/main.html.erb]: Minor: I'd prefer the error links to be updated to point at hyphenated IDs.

File: app/views/child_benefit_tax/main.html.erb [hunk 2]
```diff
@@ -125,15 +125,15 @@
       </fieldset>
     <% end %>
 
-    <div id="children-template" style="display:none">
+    <div id="children-template" style="display:none" class="panel panel-border-wide">
```
Comment (fofr) [inline app/views/child_benefit_tax/main.html.erb]: Could you add some screens showing the affect of this change?

Comment (andysellick) [inline app/views/child_benefit_tax/main.html.erb]: Screenshots in description updated.

**Comment by nickcolley** [comment]
I've had a look at the JavaScript behaviour since I was worried there did not seem to be tests covering that behaviour (something that @ikennaokpala also mentioned in person).

When commenting out code the tests do indeed fail indicating that the current tests cover the behaviour which makes me more confident in this area.

**Comment by nickcolley** [comment]
I've tested the following combinations and confirmed the same results appear locally and live

Part years for one child
£893.20
[link](https://www.gov.uk/child-benefit-tax-calculator/main?childcare=%C2%A3123.00&children_count=1&cycle_scheme=%C2%A3123.00&gift_aid_donations=%C2%A3123.00&gross_income=%C2%A3123.00&is_part_year_claim=yes&non_employment_income=%C2%A3123.00&other_income=%C2%A3123.00&outgoing_pension_contributions=%C2%A3123.00&part_year_children_count=1&pension_contributions_from_pay=%C2%A3123.00&pensions=%C2%A3123.00&property=%C2%A3123.00&results=Calculate&retirement_annuities=%C2%A3123.00&starting_children%5B0%5D%5Bstart%5D%5Bday%5D=3&starting_children%5B0%5D%5Bstart%5D%5Bmonth%5D=6&starting_children%5B0%5D%5Bstart%5D%5Byear%5D=2013&starting_children%5B0%5D%5Bstop%5D%5Bday%5D=3&starting_children%5B0%5D%5Bstop%5D%5Bmonth%5D=3&starting_children%5B0%5D%5Bstop%5D%5Byear%5D=2016&year=2013#results)

5 children
£3,926.00
[link](https://www.gov.uk/child-benefit-tax-calculator/main?childcare=123&children_count=5&cycle_scheme=123&gift_aid_donations=123&gross_income=123&is_part_year_claim=yes&non_employment_income=123&other_income=123&outgoing_pension_contributions=123&part_year_children_count=5&pension_contributions_from_pay=123&pensions=123&property=123&results=Calculate&retirement_annuities=123&starting_children%5B0%5D%5Bstart%5D%5Bday%5D=1&starting_children%5B0%5D%5Bstart%5D%5Bmonth%5D=2&starting_children%5B0%5D%5Bstart%5D%5Byear%5D=2011&starting_children%5B0%5D%5Bstop%5D%5Bday%5D=&starting_children%5B0%5D%5Bstop%5D%5Bmonth%5D=&starting_children%5B0%5D%5Bstop%5D%5Byear%5D=&starting_children%5B1%5D%5Bstart%5D%5Bday%5D=2&starting_children%5B1%5D%5Bstart%5D%5Bmonth%5D=1&starting_children%5B1%5D%5Bstart%5D%5Byear%5D=2011&starting_children%5B1%5D%5Bstop%5D%5Bday%5D=&starting_children%5B1%5D%5Bstop%5D%5Bmonth%5D=&starting_children%5B1%5D%5Bstop%5D%5Byear%5D=&starting_children%5B2%5D%5Bstart%5D%5Bday%5D=2&starting_children%5B2%5D%5Bstart%5D%5Bmonth%5D=2&starting_children%5B2%5D%5Bstart%5D%5Byear%5D=2011&starting_children%5B2%5D%5Bstop%5D%5Bday%5D=&starting_children%5B2%5D%5Bstop%5D%5Bmonth%5D=&starting_children%5B2%5D%5Bstop%5D%5Byear%5D=&starting_children%5B3%5D%5Bstart%5D%5Bday%5D=1&starting_children%5B3%5D%5Bstart%5D%5Bmonth%5D=1&starting_children%5B3%5D%5Bstart%5D%5Byear%5D=2011&starting_children%5B3%5D%5Bstop%5D%5Bday%5D=&starting_children%5B3%5D%5Bstop%5D%5Bmonth%5D=&starting_children%5B3%5D%5Bstop%5D%5Byear%5D=&starting_children%5B4%5D%5Bstart%5D%5Bday%5D=1&starting_children%5B4%5D%5Bstart%5D%5Bmonth%5D=1&starting_children%5B4%5D%5Bstart%5D%5Byear%5D=2011&starting_children%5B4%5D%5Bstop%5D%5Bday%5D=&starting_children%5B4%5D%5Bstop%5D%5Bmonth%5D=&starting_children%5B4%5D%5Bstop%5D%5Byear%5D=&year=2016#results)

No part years
£1,076.40
[link](https://www.gov.uk/child-benefit-tax-calculator/main?childcare=%C2%A3123.00&children_count=1&cycle_scheme=%C2%A3123.00&gift_aid_donations=%C2%A3123.00&gross_income=%C2%A3123.00&is_part_year_claim=no&non_employment_income=%C2%A3123.00&other_income=%C2%A3123.00&outgoing_pension_contributions=%C2%A3123.00&pension_contributions_from_pay=%C2%A3123.00&pensions=%C2%A3123.00&property=%C2%A3123.00&results=Calculate&retirement_annuities=%C2%A3123.00&year=2016#results)

Tax to pay
£438.10 --- £438.00
[link](http://www.gov.uk/child-benefit-tax-calculator/main?childcare=%C2%A3123.00&children_count=3&cycle_scheme=%C2%A3123.00&gift_aid_donations=%C2%A3123.00&gross_income=%C2%A312300000.00&is_part_year_claim=yes&non_employment_income=%C2%A3123.00&other_income=%C2%A3123.00&outgoing_pension_contributions=%C2%A3123.00&part_year_children_count=3&pension_contributions_from_pay=%C2%A3123.00&pensions=%C2%A3123.00&property=%C2%A3123.00&results=Calculate&retirement_annuities=%C2%A3123.00&starting_children%5B0%5D%5Bstart%5D%5Bday%5D=1&starting_children%5B0%5D%5Bstart%5D%5Bmonth%5D=2&starting_children%5B0%5D%5Bstart%5D%5Byear%5D=2011&starting_children%5B0%5D%5Bstop%5D%5Bday%5D=&starting_children%5B0%5D%5Bstop%5D%5Bmonth%5D=&starting_children%5B0%5D%5Bstop%5D%5Byear%5D=&starting_children%5B1%5D%5Bstart%5D%5Bday%5D=2&starting_children%5B1%5D%5Bstart%5D%5Bmonth%5D=1&starting_children%5B1%5D%5Bstart%5D%5Byear%5D=2011&starting_children%5B1%5D%5Bstop%5D%5Bday%5D=&starting_children%5B1%5D%5Bstop%5D%5Bmonth%5D=&starting_children%5B1%5D%5Bstop%5D%5Byear%5D=&starting_children%5B2%5D%5Bstart%5D%5Bday%5D=5&starting_children%5B2%5D%5Bstart%5D%5Bmonth%5D=2&starting_children%5B2%5D%5Bstart%5D%5Byear%5D=2015&starting_children%5B2%5D%5Bstop%5D%5Bday%5D=&starting_children%5B2%5D%5Bstop%5D%5Bmonth%5D=&starting_children%5B2%5D%5Bstop%5D%5Byear%5D=&year=2012#results)

**Pull request merged**